### PR TITLE
update stagenet script

### PR DESCRIPTION
**Pull request by noot**
update script so that it's for running a single node on stagenet, fix typo

### CODE REVIEW

File: docs/stagenet.md [hunk 1]
```diff
@@ -14,13 +14,13 @@ The atomic swap daemon requires a connection to a monero-wallet-rpc process conn
 
 For Linux 64-bit, you can do:
 ```bash
-curl -L https://downloads.getmonero.org/cli/linux64 > monero.tar.bz2
-tar xjvf monero.tar.bz2
+bash scripts/install-monero-linux.sh
```
Comment (dimalinux) [inline docs/stagenet.md:17]: The script has execute permissions (which git respects/maintains) and it starts with `#!/bin/bash`. Any reason to not execute it directly?  i.e. `./scripts/install-monero-linux.sh`?

File: scripts/setup-stagenet.sh [hunk 2]
```diff
@@ -7,17 +7,14 @@ echo "starting monerod..."
 sleep 5
 
 echo "starting monero-wallet-rpc on port 18083..."
-nohup ./monero-bin/monero-wallet-rpc --rpc-bind-port 18083 --disable-rpc-login --wallet-dir ./bob-test-keys --stagenet --trusted-daemon
+nohup ./monero-bin/monero-wallet-rpc --rpc-bind-port 18083 --disable-rpc-login --wallet-dir ./node-keys --stagenet --trusted-daemon &> monero-wallet-cli.log &
```
Comment (dimalinux) [inline scripts/setup-stagenet.sh:10]: Should we try the `--detach` and `--log-file` flags to `monero-wallet-rpc`?

**Review by dimalinux — APPROVED** [review]

File: docs/stagenet.md (hunk 1 above)
Comment (noot) [reply docs/stagenet.md]: good point, updated!

File: scripts/setup-stagenet.sh (hunk 2 above)
Comment (noot) [reply scripts/setup-stagenet.sh:10]: that would be nice, didn't realize it had a `--detach` flag as well. I can update all of the scripts in a follow-up

**Pull request merged**